unittests/unittest_utils.py

```diff
@@ -173,5 +173,36 @@ def create_random_string(length=8):
     return "".join(random.choice(char_set) for x in range(length))
 
 
+# checks line by line that parens and quotations are balanced
+def balanced_text(myText):
```

Comment:
Use a docstring for the function

PS Function is OK for this ticket. I sent an inquiry for a possible linter provided by the condor team. Maybe we ask also this Friday.

Reply:
done

Reply:
done